src/constants/lists.ts

```diff
@@ -5,13 +5,16 @@ const CMC_ALL_LIST = 'https://api.coinmarketcap.com/data-api/v3/uniswap/all.json
 const COINGECKO_LIST = 'https://tokens.coingecko.com/uniswap/all.json'
 const COMPOUND_LIST = 'https://raw.githubusercontent.com/compound-finance/token-list/master/compound.tokenlist.json'
 const GEMINI_LIST = 'https://www.gemini.com/uniswap/manifest.json'
-export const ARBITRUM_LIST = 'https://bridge.arbitrum.io/token-list-42161.json'
 const KLEROS_LIST = 't2crtokens.eth'
-export const OPTIMISM_LIST = 'https://static.optimism.io/optimism.tokenlist.json'
 const ROLL_LIST = 'https://app.tryroll.com/tokens.json'
 const SET_LIST = 'https://raw.githubusercontent.com/SetProtocol/uniswap-tokenlist/main/set.tokenlist.json'
 const WRAPPED_LIST = 'wrapped.tokensoft.eth'
 
+export const OPTIMISM_LIST = 'https://static.optimism.io/optimism.tokenlist.json'
+export const ARBITRUM_LIST = 'https://bridge.arbitrum.io/token-list-42161.json'
+
+export const CELO_LIST = 'https://raw.githubusercontent.com/celo-org/celo-token-list/main/celo.tokenlist.json'
```

Comment:
hosting content like this on github is *technically* against ToS. It'll prob be fine, but if you can host it yourself that's better.

Reply:
does celo make the token list available elsewhere? do they document it anywhere? if we know it's against ToS we should not be doing it.

Reply:
Can you please statically host it and use that link here?

Reply:
Sure thing, on it now

Reply:
done